docs/Acceptable-Formulae.md

```diff
@@ -78,9 +78,9 @@ etc.
 
 If not, then put bindings in the formula they bind to. This is more
 useful to people. Just install the stuff! Having to faff around with
-foo-ruby foo-perl etc. sucks.
+foo-ruby, foo-perl etc. sucks.
 
-### Niche (or self-submitted) Stuff<a name="Niche_Stuff"></a>
+### Niche (or self-submitted) stuff<a name="Niche_Stuff"></a>
```

Comment:
Kramdown auto-generates IDs for each title, which can be linked to in lieu of `name` attributes, e.g. http://docs.brew.sh/Querying-Brew.html#examples